### ISSUE TITLE
PP - PL relation problems - what do we do with PP.description field?

### ISSUE THREAD

**Issue by Nuanda** [eSpectrum-IT/brassica#180]
This is probably false alarm, but please make the following sanity check (to be sure we haven't dropped sth important).

List PP.description for all PP records - most of the time you'll find there information about number of PLs expected to be connected to the given PP. Now, many of these PPs are not connected to any PL (via PPL table) regardless what the `description` field says.

Please simply make sure, that in such cases the description is wrong, and not that we have dropped some PP - PL relations we shouldn't have.

**Comment by nowakowski** [comment]
I verified this issue for three randomly picked populations:
  - BolTBDH_01 (description claims 152 PLs)
  - BraCKDH_02 (42 PLs)
  - BolAGSL_02 (67 PLs)

I can confirm that these populations were not bound to any plant lines via `plant_population_lists` in the original DB - i.e. we didn't lose any assignments in the conversion process. Looks like a curation issue.

**Comment by Nuanda** [comment]
@wjurkowski @sarahcayling A lot of PlantPopulations claim, in their description column, that they have a certain number of PlantLines connected to them. However, in many cases, they have none. Perhaps these PlantLines are there, but they were never (properly) connected in the CS data. So, eventually they are not connected in BIP - i.e. you will not be able to navigate from them to their associated PlantLines (as you are able with those PlantPopulations that do have correct relation to PLs).

Now, the question is: *do we show the description column in the PP table?* I found it might be valuable so I included it in the table, yet now, seeing so many strikingly wrong information there, I'm not certain anymore.
